airflow/models/__init__.py

```diff
@@ -3249,6 +3259,17 @@ def __exit__(self, _type, _value, _tb):
 
     # /Context Manager ----------------------------------------------
 
+    @property
+    def default_view(self):
+        if self._default_view is None:
+            return configuration.conf.get('webserver', 'dag_default_view').lower()
+        else:
+            return self._default_view
+
+    def get_default_view(self):
+        """This is only there for backward compatible jina2 templates"""
```

Comment:
I think I'm missing something here. Why can't the templates continue using the `default_view` property? Also, s/jina2/jinja2/.

Reply:
This was to do with https://issues.apache.org/jira/browse/AIRFLOW-3562.
Currently DAG is still used in other templates, can't change all those at once. Once all views has switched to DagModel this method can be removed again.

Reply:
ps: fixing typo ;)